src/FunctionsP/GetRanges.py

```diff
@@ -0,0 +1,27 @@
+def get_ranges(lst_1=[int(el) for el in input().split()]):
```

Comment:
put input to function definitions default value isn't a good idea. 
more of that it would work once